examples/sql_bank/main.go

```diff
@@ -202,20 +206,28 @@ func main() {
 
 	verifyBank(db)
 
-	var lastNumTransfers uint64
 	lastNow := time.Now()
+	readings := make(chan measurement, 10000)
 
 	for i := 0; i < *concurrency; i++ {
-		go moveMoney(db)
+		go moveMoney(db, readings)
 	}
 
 	for range time.NewTicker(*balanceCheckInterval).C {
 		now := time.Now()
 		elapsed := time.Since(lastNow)
-		numTransfers := atomic.LoadUint64(&numTransfers)
-		log.Infof("%d transfers were executed at %.1f/second.", (numTransfers - lastNumTransfers), float64(numTransfers-lastNumTransfers)/elapsed.Seconds())
-		verifyBank(db)
-		lastNumTransfers = numTransfers
 		lastNow = now
+		transfers := len(readings)
```

Comment:
Not sure that can be used here. Range over channel will block until the channel is closed.